packages/@aws-cdk/aws-ses/lib/configuration-set-event-destination.ts

```diff
@@ -0,0 +1,279 @@
+   */
+  RENDERING_FAILURE = 'renderingFailure',
+
+  /**
+   * The email couldn't be delivered to the recipient’s mail server because a temporary
+   * issue occurred. Delivery delays can occur, for example, when the recipient's inbox
+   * is full, or when the receiving email server experiences a transient issue.
+   */
+  DELIVERY_DELAY = 'deliveryDelay',
+
+  /**
+   * The email was successfully delivered, but the recipient updated their subscription
+   * preferences by clicking on an unsubscribe link as part of your subscription management.
+   */
+  SUBSCRIPTION = 'subscription'
+}
+
+/**
+ * A CloudWatch dimension upon which to categorize your emails
+ */
+export interface CloudWatchDimension {
+  /**
+   * The place where Amazon SES finds the value of a dimension to publish to
+   * Amazon CloudWatch.
+   */
+  readonly source: CloudWatchDimensionSource;
+
+  /**
+   * The name of an Amazon CloudWatch dimension associated with an email sending metric.
+   */
+  readonly name: string;
+
+  /**
+   * The default value of the dimension that is published to Amazon CloudWatch
+   * if you do not provide the value of the dimension when you send an email.
+   */
+  readonly default: string;
+}
+
+/**
+ * Source for CloudWatch dimension
+ */
+export enum CloudWatchDimensionSource {
+  /**
+   * Amazon SES retrieves the dimension name and value from a header in the email.
+   *
+   * Note: You can't use any of the following email headers as the Dimension Name:
+   * `Received`, `To`, `From`, `DKIM-Signature`, `CC`, `message-id`, or `Return-Path`.
+   */
+  EMAIL_HEADER = 'emailHeader',
+
+  /**
+   * Amazon SES retrieves the dimension name and value from a tag that you specified in a link.
+   *
+   * @see https://docs.aws.amazon.com/ses/latest/dg/faqs-metrics.html#sending-metric-faqs-clicks-q5
+   */
+  LINK_TAG = 'linkTag',
+
+  /**
+   * Amazon SES retrieves the dimension name and value from a tag that you specify by using the
+   * `X-SES-MESSAGE-TAGS` header or the Tags API parameter.
+   *
+   * You can also use the Message Tag value source to create dimensions based on Amazon SES auto-tags.
+   * To use an auto-tag, type the complete name of the auto-tag as the Dimension Name. For example,
+   * to create a dimension based on the configuration set auto-tag, use `ses:configuration-set` for the
+   * Dimension Name, and the name of the configuration set for the Default Value.
+   *
+   * @see https://docs.aws.amazon.com/ses/latest/dg/event-publishing-send-email.html
+   * @see https://docs.aws.amazon.com/ses/latest/dg/monitor-using-event-publishing.html#event-publishing-how-works
+   */
+  MESSAGE_TAG = 'messageTag',
+}
+
+/**
+ * A configuration set event destination
+ */
+export class ConfigurationSetEventDestination extends Resource implements IConfigurationSetEventDestination {
+  /**
+   * Use an existing configuration set
+   */
+  public static fromConfigurationSetEventDestinationId(
```

Comment:
Is there a case to also have a `fromConfigurationSetEventDestinationName`?

Reply:
The `id` is the name I think but the doc is not really clear...

![image](https://user-images.githubusercontent.com/12623249/229276777-f98a90a9-7c4b-4cb3-b397-bf5dca63c4fa.png)


Anyway I don't think this resource will be referenced by another CF resource, even the `fromConfigurationSetEventDestinationId()` will not be used I think... (but makes the linter happy)